README.md

```diff
@@ -80,6 +80,36 @@ various sources of help. If you want a specific starting point there are
 on [porting pure Python modules](http://docs.python.org/3/howto/pyporting.html)
 and [extension modules](http://docs.python.org/3/howto/cporting.html).
 
+
+## Use it as a pre-commit hook
+
+
+Install Pre-commit
+
+```
+pip install pre-commit==1.11.1
+pre-commit install
+```
+
+It is best used along with [pre-commit](https://pre-commit.com/) You can use it along with pre-commit by adding the following hook in your ``.pre-commit-config.yaml`` file.
+
+    - repo: https://github.com/brettcannon/caniusepython3
+      sha: 'b97a4cd7'
+      hooks:
+          - id: caniusepython3
+            files: requirements.txt  # Update to match your requirements files accordingly.
```

Comment:
@brettcannon yes, they would have to override this accordingly to how their requirements files are named.